src/main/java/com/jwplayer/southpaw/util/ByteArray.java

```diff
@@ -31,6 +31,7 @@
  * Wrapper class for byte arrays so we can use them as keys in maps.
  */
 public class ByteArray implements Comparable<ByteArray>, Serializable {
+    private static final long serialVersionUID = -6277178128299377659L;
```

Comment:
i cant find any references to this variable. If nothing uses it then maybe we should delete it?

Reply:
nevermind. read the docs here
https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/io/Serializable.html